yt_dlp/extractor/rule34video.py

```diff
@@ -52,6 +76,38 @@ def _real_extract(self, url):
         title = self._html_extract_title(webpage)
         thumbnail = self._html_search_regex(r'preview_url:\s+\'([^\']+)\'', webpage, 'thumbnail', default=None)
         duration = self._html_search_regex(r'"icon-clock"></i>\s+<span>((?:\d+:?)+)', webpage, 'duration', default=None)
+        like_count = str_to_int(remove_end(get_element_by_class('voters count', webpage), ' likes'))
+        comment_count = str_to_int(self._html_search_regex(
+            r'<a[^>]+href="#tab_comments"[^>]*>[^(]+\(([^)]+)', webpage, 'comments', default=None))
```

Comment:
Maybe this is a bit cleaner? Not sure
```suggestion
        comment_count = int_or_none(self._search_regex(
            r'Comments\s+\((\d+)\)', get_element_by_attribute('href', '#tab_comments', webpage),
            'comment count', fatal=False))
```
`get_element_by_attribute` and `int_or_none` need to be imported.

Reply:
Done.